src/components/PostsList.tsx

```diff
@@ -1,86 +1,72 @@
 import React from 'react';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  userPosts: Post[];
+  handleShowComment: (post: Post) => void;
+  showSideBar: number | null;
+  setShowSideBar: (id: number | null) => void;
+  setShowCommentForm: (item: boolean) => void;
+};
```

Comment:
Make sure you described objects in propTypes. Although you're using TypeScript, propTypes can be useful for checking types at runtime, especially if your code will be used in a JavaScript context.